fieldservice_fsm_person_salary_extended/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_fsm_salary_user,fsm.salary.user,model_fsm_salary,fieldservice.group_fsm_user,1,1,0,0
```

Comment:
Is it okay for a user to be able to create record? I think it shouldn't

Reply:
A FSM User cannot create a FSM order, but can modify it, @ChristianSantamaria could you verify if a FSM User has "Edit" button enabled? In that case, you're right, modifying a FSM order implies working link addition => creation permission for this new model

Reply:
You are right, it cannot create but it does have the edit button